opensearch-api/lib/opensearch/api/actions/cat/all_pit_segments.rb

```diff
@@ -0,0 +1,46 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+#
+# Modifications Copyright OpenSearch Contributors. See
+# GitHub history for details.
+
+module OpenSearch
+  module API
+    module Cat
+      module Actions
+        # Retrieves info of all PIT segments
+        #
+        # @option arguments [String] :format a short version of the Accept header, e.g. json, yaml
+        # @option arguments [List] :h Comma-separated list of column names to display
+        # @option arguments [Boolean] :help Return help information
+        # @option arguments [List] :s Comma-separated list of column names or column aliases to sort by
+        # @option arguments [Boolean] :v Verbose mode. Display column headers
+        # @option arguments [Hash] :headers Custom HTTP headers
+        def all_pit_segments(arguments = {})
+          arguments = arguments.clone
+          headers = arguments.delete(:headers) || {}
+
```

Comment:
Double space, ... we should really add rubocop.

Reply:
I miss Rubocop. Will incorporate along with generated Ruby Client.

Reply:
https://github.com/opensearch-project/opensearch-ruby/pull/165